flang-rt/CMakeLists.txt

```diff
@@ -334,8 +334,8 @@ if (LLVM_INCLUDE_EXAMPLES)
 endif ()
 
 if (FLANG_RT_INCLUDE_TESTS)
-  add_subdirectory(unittests)
   add_subdirectory(test)
+  add_subdirectory(unittests)
```

Comment:
This is now second because `test` will do the lit config file setup, which `unittests` then uses?